tests/phpunit/tests/rest-api/rest-schema-validation.php

```diff
@@ -349,10 +349,25 @@ public function test_object_or_string() {
 		$this->assertWPError( rest_validate_value_from_schema( array( 'raw' => array( 'a list' ) ), $schema ) );
 	}
 
-	/*
+	public function rest_property_provider() {
+		return array(
+			array(
+				(object) array(
```

Comment:
These don't need to be casted to `object`. In essentially all cases, the API passes associative arrays. See `WP_REST_Request::parse_json_params`.